scripts/test

```diff
@@ -61,9 +64,9 @@ compose() {
 
   # Run tests
   if [ "${debug}" = "true" ]; then
-    compose run --service-ports ${service} ${command}
+    compose run --service-ports --rm ${service} ${command}
```

Comment:
Does the --rm flag mean remove?

Reply:
It automatically clears up the container once it's completed.

So for example, the script runs a container with the tests in and then exists when the tests are complete.

It will do that every time you run the script.

Without the `--rm` argument, your machine will end up full of exited containers.

So keeps dev's machines neat and tidy.